Libraries/react-native/react-native.js

```diff
@@ -86,6 +86,7 @@ var ReactNative = {
   get StyleSheet() { return require('StyleSheet'); },
   get TimePickerAndroid() { return require('TimePickerAndroid'); },
   get UIManager() { return require('UIManager'); },
+  get Vibration() { return require('Vibration'); },
```

Comment:
Please also update `react-native.flow.js`. It's a copy of this file for Flow as Flow doesn't understand getters well yet.